glaciers/glaciers.ipynb

```diff
@@ -351,26 +244,29 @@
     ")\n",
     "\n",
     "template.sidebar.extend([\n",
-    "    pn.pane.PNG(oggm_url, width=250),\n",
-    "    pn.pane.Markdown(instruction, width=250, height=100),\n",
-    "    pn.pane.PNG(pn_url, width=200, align='center')\n",
+    "    oggm_logo,\n",
+    "    instruction,\n",
+    "    panel_logo,\n",
     "])\n",
     "\n",
-    "template.main.append(\n",
-    "    (geomap + elevation + temperature + precipitation).cols(2)\n",
-    ")\n",
+    "template.main.append(plots)\n",
     "\n",
-    "template.servable();"
+    "template.show();"
```

Comment:
You need to use `.servable()` to correctly serve an app with `panel serve appfile`. `.show()` is good for development purposes and local usages.

```suggestion
    "template.servable();"
```

Reply:
Sorry, that was an oversight on my part.